.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why did you add this file? We have our configs in packages and we use configuration from `eslint-config-custom`.